### PR TITLE
Add more events details supports for actions

### PR DESCRIPTION
**Pull request by lunny**
#21937 implemented only basic events based on name because of `act`'s limitation. So I sent a PR to parse all possible events details in https://gitea.com/gitea/act/pulls/11 and it merged. The ref documentation is https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows

This PR depends on that and make more detail responses for `push` events and `pull_request` events. And it lefts more events there for future PRs.

### CODE REVIEW

**Review by wolfogre — APPROVED** [review]

File: modules/actions/workflows.go [hunk 1]
```diff
@@ -63,13 +66,156 @@ func DetectWorkflows(commit *git.Commit, event webhook_module.HookEventType) (ma
...
+		return true
+	}
+
+	switch triggedEvent {
+	case webhook_module.HookEventCreate:
+		fallthrough
+	case webhook_module.HookEventDelete:
+		fallthrough
+	case webhook_module.HookEventFork:
+		log.Error("unsupported event %q", triggedEvent.Event())
```
Comment (zeripath) [inline modules/actions/workflows.go]: Perhaps this should just be Warn?

File: modules/actions/workflows.go [hunk 2]
```diff
@@ -63,13 +66,156 @@ func DetectWorkflows(commit *git.Commit, event webhook_module.HookEventType) (ma
...
+	case webhook_module.HookEventWiki:
+		fallthrough
+	case webhook_module.HookEventRepository:
+		fallthrough
+	case webhook_module.HookEventRelease:
+		fallthrough
+	case webhook_module.HookEventPackage:
+		fallthrough
+	default:
+		log.Error("unsupported event %q", triggedEvent.Event())
```
Comment (zeripath) [inline modules/actions/workflows.go]: Similarly should this just be warn?

File: modules/actions/workflows.go (hunk 1 above)
Comment (wolfogre) [inline modules/actions/workflows.go]:
```suggestion
		log.Warn("unsupported event %q", triggedEvent.Event())
```

File: modules/actions/workflows.go (hunk 2 above)
Comment (wolfogre) [inline modules/actions/workflows.go]:
```suggestion
		log.Warn("unsupported event %q", triggedEvent.Event())
```

**Review by zeripath — APPROVED** [review]

**Pull request merged**